checks.d/pgbouncer.py

```diff
@@ -0,0 +1,167 @@
+"""Pgbouncer check
+
+Collects metrics from the pgbouncer database.
+"""
+from checks import AgentCheck, CheckException
+from collections import OrderedDict
+
+import psycopg2 as pg
+class ShouldRestartException(Exception): pass
+
+class PgBouncer(AgentCheck):
+    """Collects metrics from pgbouncer
+    """
+    SOURCE_TYPE_NAME = 'pgbouncer'
+    RATE = AgentCheck.rate
+    GAUGE = AgentCheck.gauge
+
+    STATS_METRICS = {
+        'descriptors': [
+            ('database', 'db'),
+        ],
+        'metrics': OrderedDict([
+            ('total_requests', ('pgbouncer.stats.total_requests', RATE)),
+            ('total_received', ('pgbouncer.stats.total_received', RATE)),
+            ('total_sent', ('pgbouncer.stats.total_sent', RATE)),
+            ('total_query_time', ('pgbouncer.stats.total_query_time', RATE)),
```

Comment:
This becomes a unit-less measure, as it's time divided by time.

It is the amount of time spent in the past sampling interval processing queries?